plugin/ui/completion.py

```diff
@@ -274,3 +295,94 @@ def show(self) -> None:
     @staticmethod
     def hide(view: sublime.View) -> None:
         mdpopups.hide_popup(view)
+
+
+class _PhantomCompletion(_BaseCompletion):
+    COPILOT_PHANTOM_COMPLETION = "copilot_phantom_completion"
+    PHANTOM_TEMPLATE = """
+    <body id="copilot-completion">
+        <style>
+            body {{
+                color: #808080;
```

Comment:
Should we use `comment` color for this? 

E.g.,
```py
>>> view.style_for_scope('comment')
{'bold': False, 'foreground': '#616161', 'italic': False, 'selection_foreground': '#757575', ...}
```

Reply:
I don't know, I took this colour from [the neovim plugin](https://github.com/github/copilot.vim/blob/release/plugin/copilot.vim#L14-L20) and it works ok for both light and dark modes.

What I would do I would add a note on how to customize it, but, IMO, it sounds like a reasonable default

Reply:
If users don't complain in the neovim plugin, I guess we don't have to do that until many of them complain.